selfdrive/loggerd/omx_encoder.cc

```diff
@@ -343,16 +343,6 @@ void OmxEncoder::callback_handler(OmxEncoder *e) {
 
     e->to_write.push(new_buffer);
 
-#ifdef QCOM2
```

Comment:
Did you verify that this is not needed to properly hand back the buffer to OMX? It was introduced to "Fix random encoder lockups" here: https://github.com/commaai/openpilot/pull/2707. In January we switched to a different Venus firmware, which might have been the real fix: https://commadotai.slack.com/archives/C0155QM7057/p1611277394048100.

Probably best to not have functional changes in this refactor PR, and move this functional change into a separate PR so it's more obvious if we notice any issues later.

Reply:
oh, i didn't realize that's why this was done! i thought it was for the ffmpeg remuxer, and i fixed that bug